editor.planx.uk/src/@planx/graph/__tests__/formatOps.test.ts

```diff
@@ -0,0 +1,269 @@
+import { formatOps,Graph } from "../index";
+
+describe("Update operations", () => {
+  test("Updating a single property of a node", () => {
+    const ops = [
+      {
+        p: ["FW5G3EMBI3", "data", "text"],
+        oi: "Which vegetables?",
+        od: "Which fruits?",
+      },
+    ];
+
+    expect(formatOps(flowWithChecklist, ops)).toEqual([
+      'Updated Checklist text from "Which fruits?" to "Which vegetables?"',
```

Comment:
It feels like helpful context to print the value of the updated data prop here, but it's a big difference between showing how `text` or `title` change than it is, for example, help text - we may want to consider making a sort of "black list" of data props where we'd only show the name `"Updated moreInfo"` _without_ the values? Could also incorporate something like nesting changes list within accordion drawers in side panel to address?

I plan to leave in initially for explicitness, but would expect this feedback to come up pretty early in testing !